packages/css-data/src/__generated__/property-value-descriptions.ts

```diff
@@ -0,0 +1,4574 @@
+  cursor:
+    "Sets the type of cursor to be displayed when hovering over an element",
+  direction: "Sets the text direction",
+  display: "Defines how an element should be displayed",
+  emptyCells: "Controls the rendering of empty cells in a table",
+  filter: "Applies visual effects (like blur or color shift) to an element",
+  flexBasis: "Specifies the initial size of a flex item",
+  flexDirection: "Defines the direction of the flex container",
+  flexGrow:
+    "Specifies the ability of a flex item to grow to fill available space",
+  flexShrink: "Specifies the ability of a flex item to shrink if necessary",
+  flexWrap:
+    "Specifies whether flex items are forced onto one line or can wrap onto multiple lines",
+  float: "Specifies the horizontal alignment of an element",
+  fontFamily: "Sets the font family for text",
+  fontFeatureSettings:
+    "Enables control over advanced typographic features in OpenType fonts",
+  fontKerning: "Controls the spacing between glyphs in a font",
+  fontLanguageOverride: "Overrides the language of the font",
+  fontOpticalSizing:
+    "Enables consistent sizing of fonts for optimal readability",
+  fontVariationSettings:
+    "Enables fine control over OpenType font features such as kerning and ligatures.",
+  fontSize: "Sets the size of the font",
+  fontSizeAdjust: "Adjusts the font size to optimize legibility",
+  fontStretch: "Stretches or compresses the font horizontally",
+  fontStyle: "Sets the style of the font (normal, italic, oblique)",
+  fontSynthesis: "Controls the synthesis of fonts",
+  fontVariant: "Sets the variant of the font (normal, small-caps)",
+  fontVariantAlternates: "Controls alternate glyphs for certain characters",
+  fontVariantCaps: "Controls capitalization of text",
+  fontVariantEastAsian: "Controls alternate glyphs for East Asian scripts",
+  fontVariantLigatures: "Controls which ligatures are used in the font",
+  fontVariantNumeric: "Controls display of numeric characters",
+  fontVariantPosition: "Controls the position of the font variant",
+  fontWeight: "Sets the weight (boldness) of the font",
+  forcedColorAdjust: "Controls how colors are adjusted for accessibility",
+  gridAutoColumns: "Sets the size of auto-generated columns in a grid",
+  gridAutoFlow: "Controls how items are placed in auto-generated grid tracks",
+  gridAutoRows: "Sets the height of an element",
+  gridColumnEnd: "Sets the character used to hyphenate words",
+  gridColumnStart: "Sets whether words can be hyphenated",
+  gridRowEnd: "Sets the orientation of an image",
+  gridRowStart: "Sets the algorithm used to render an image",
+  gridTemplateAreas: "Sets the resolution of an image",
+  gridTemplateColumns: "Sets the size of the first letter in a block of text",
+  gridTemplateRows: "Sets the starting position of a grid item's column",
+  hangingPunctuation: "Sets the ending position of a grid item's column",
+  height: "Sets the starting position of a grid item's row",
```

Comment:
well this one is a complete fail :)

Reply:
the good thing is that we can tweak this manually. If we add properties to the source file that we use to generate the script will generate only new entries (unless you run it with --force)

Reply:
so we can add the values here directly?

Reply:
technically yes

Reply:
alright, going to do some edits now